### PR TITLE
fix(highlight): fix owning tree highlighting

### PR DESCRIPTION
**Pull request by steelsojka**
Overlooked this one thing.

### CODE REVIEW

File: lua/nvim-treesitter-playground/internal.lua [hunk 1]
```diff
@@ -48,9 +48,13 @@ local function get_node_at_cursor()
...
 
   for _, tree in ipairs(owning_lang_tree:trees()) do
-    result = tree:root():named_descendant_for_range(lnum - 1, col, lnum - 1, col)
+    local range = {lnum - 1, col, lnum - 1, col}
 
-    if result then return result end
+    if utils.node_contains(tree:root(), range) then
+      result = tree:root():descendant_for_range(unpack(range))
+
+      if result then return result end
```
Comment (theHamsta) [inline lua/nvim-treesitter-playground/internal.lua:56]: Why not just `return result`

**Review by theHamsta — APPROVED** [review]

File: lua/nvim-treesitter-playground/internal.lua (hunk 1 above)
Comment (steelsojka) [reply lua/nvim-treesitter-playground/internal.lua:56]: I wasn't sure if this always returned a node.... but I think it might?

Comment (theHamsta) [reply lua/nvim-treesitter-playground/internal.lua:56]: Sure it's in a loop. My fault...

Comment (steelsojka) [reply lua/nvim-treesitter-playground/internal.lua:56]: oh I forgot about that as well lol

**Pull request merged**